setup.py

```diff
@@ -167,9 +167,9 @@ def get_version(_mod_root):
 
 
 # ------------------------------------------------------------------------------
-# check python version, should be >= 3.6
+# check python version, should be >= 3.7
 if sys.hexversion < 0x03060000:
```

Comment:
`if sys.hexversion < 0x03070000:`

Reply:
Ah, I knew we had this somewhere - overlooked...  Thanks!